Makefile

```diff
@@ -249,52 +228,34 @@ dist/calico-felix/calico-felix: bin/calico-felix
 update-tools:
 	go get -u github.com/Masterminds/glide
 	go get -u github.com/onsi/ginkgo/ginkgo
-	go get -u github.com/wadey/gocovmerge
 
 # Run go fmt on all our go files.
 .PHONY: go-fmt
 go-fmt:
-	$(MAKE) calico-build/golang
-	$(DOCKER_RUN_RM) -w /code/ calico-build/golang sh -c 'glide nv | xargs go fmt'
+	$(DOCKER_GO_BUILD) sh -c 'glide nv | xargs go fmt'
+
+licensecheck/dependency-licenses.txt: vendor/.up-to-date
+	$(DOCKER_GO_BUILD) sh -c 'licenses . > licensecheck/dependency-licenses.txt'
 
 .PHONY: ut
 ut combined.coverprofile: vendor/.up-to-date $(GO_FILES)
 	@echo Running Go UTs.
-	$(MAKE) calico-build/golang
-	mkdir -p .go-pkg-cache
-	$(DOCKER_RUN_RM) \
-	    --net=host \
-	    -v $${PWD}:/go/src/github.com/projectcalico/felix:rw \
-	    -v $${PWD}/.go-pkg-cache:/go/pkg/:rw \
-	    -w /go/src/github.com/projectcalico/felix \
-	    calico-build/golang \
-	    ./utils/run-coverage
+	$(DOCKER_GO_BUILD) ./utils/run-coverage
+
+.PHONY: check-licenses
+check-licenses: licensecheck/dependency-licenses.txt
+	@echo Checking dependency licenses
+	$(DOCKER_GO_BUILD) ginkgo licensecheck
 
 .PHONY: ut-no-cover
-ut-no-cover: vendor/.up-to-date $(GO_FILES)
+ut-no-cover: vendor/.up-to-date $(GO_FILES) licensecheck/dependency-licenses.txt
```

Comment:
I believe dependency-licenses.txt should not now be a dependency of the UT targets - right?